site/class-1-load/sebasschmidt/index.html

```diff
@@ -0,0 +1,58 @@
+<!doctype html>
+<meta charset=utf8>
+<!--Update the title with your username:-->
+<title>@sebasschmidt</title>
+<meta content=width=device-width,initial-scale=1 name=viewport>
+<link rel="stylesheet" type="text/css" href="index.css">
+<!--Link to d3 here:-->
```

Comment:
Could you remove the comments in your HTML? You can do that by clicking the pencil icon in the top right corner of this file here.